.env

```diff
@@ -27,3 +33,13 @@ PRICES_SERVICE_URI=prices.mainnet.teritori.com:443
 
 DISPLAYED_NETWORKS_IDS=teritori,teritori-testnet,ethereum,ethereum-goerli,cosmos-hub,juno,osmosis,solana
 SELECTABLE_NETWORKS_IDS=teritori,teritori-testnet,ethereum,ethereum-goerli
+
+DAO_CW20_CODE_ID=99
+DAO_FACTORY_CODE_ID=100
+DAO_CORE=101
+DAO_PRE_PROPOSE_SINGLE=102
+DAO_PROPOSAL_SINGLE=103
+DAO_VOTING_CW20_STAKED=104
+DAO_CW20_STAKE=105
+TERITORI_DAO_FACTORY_CONTRACT_ADDRESS=tori1r29chp8ufwgx9u3wr4sfk050aardhkzwve7nht6y06gvlaqutr3qt83z6r
```

Comment:
all these vars should come from network config

Reply:
k